aws/resource_aws_route_table.go

```diff
@@ -353,6 +353,11 @@ func resourceAwsRouteTableUpdate(d *schema.ResourceData, meta interface{}) error
 				opts.NatGatewayId = aws.String(s)
 			}
 
+			// #69 - just do it! AND don't complain on subsequent runs that it needs to _change_ :/
```

Comment:
What does this `#69` stands for?
Also the comment is a bit rough, could you reword it a bit so that we know by reading what it fixes?